### PR TITLE
adds a parameter for the rln-relay content topic

### PR DESCRIPTION
**Pull request by staheri14**
Passes the `nim_waku_rln_relay_content_topic` nwaku argument defined in https://github.com/status-im/infra-nim-waku/pull/43/files.

### CODE REVIEW

File: templates/docker-compose.yml.j2 [hunk 1]
```diff
@@ -48,6 +48,7 @@ services:
       --max-connections={{ nim_waku_p2p_max_connections }}
       --store-capacity={{ nim_waku_store_capacity }}
       --dns4-domain-name={{ nim_waku_dns4_domain_name }}
+      --rln-relay-content-topic={{nim_waku_rln_relay_content_topic}}
```
Comment (jakubgs) [inline templates/docker-compose.yml.j2:51]: Shouldn't this flag be conditional depending on whether `rln-relay` protocol is enabled?

Or is there no issue if you supply flags for disabled protocols? It's just messy?

Comment (staheri14) [reply templates/docker-compose.yml.j2:52]: @jakubgs can you please let me know if the added `if` statement looks correct to you? thanks

Comment (jakubgs) [reply templates/docker-compose.yml.j2:52]: Yes, this format of `if` condition should work just fine, BUT you're not closing it. You also need an `{% endif %}`.

Also, I'd use double quotes in Jinja2 code blocks, like in the rest of the file.

Comment (staheri14) [reply templates/docker-compose.yml.j2:52]: Thanks, addressed in https://github.com/status-im/infra-role-nim-waku/pull/1/commits/87e2849a4b0e08e2b1a482ae0b9ba0a5e34e7798

**Comment by staheri14** [comment]
@jakubgs If the PR looks fine to you, can we please merge it? It seems I do not have permission to do so

File: templates/docker-compose.yml.j2 (hunk 1 above)
Comment (staheri14) [reply templates/docker-compose.yml.j2:52]: I just realized some of the variables are enclosed in single quotes, it this also necessary for `nim_waku_rln_relay_content_topic`? this key holds a string value as well @jakubgs

Comment (jakubgs) [reply templates/docker-compose.yml.j2:52]: Depends. In most cases string values in YAML can be passed without quoting, BUT if you're doing interpolation with `{{ }}` then it has to be quotes. There are also some other edge cases with special characters where strings have to be quoted. As a general rule I try to quote everywhere, but sometimes I miss things.

Comment (jakubgs) [reply templates/docker-compose.yml.j2:52]: Also, please add spaces around `nim_waku_rln_relay_content_topic` here.

Comment (staheri14) [reply templates/docker-compose.yml.j2]: Sure, done in https://github.com/status-im/infra-role-nim-waku/pull/1/commits/4c71b953d326f1f12982174eb42020faf07b865f

**Review by jakubgs — COMMENTED** [review]
It's missing an entry with a default for  `nim_waku_rln_relay_content_topic`.

**Comment by staheri14** [comment]
I guess the default `nim_waku_rln_relay_content_topic' must be defined in https://github.com/status-im/infra-role-nim-waku/blob/master/defaults/main.yml? right?

Also, wondering why a default value is needed? given that the `nim_waku_rln_relay_content_topic` option is passed only when the `rln-relay` is enabled which is only the case for the test fleets for which the `nim_waku_rln_relay_content_topic' is defined

**Comment by jakubgs** [comment]
If there is no sane default value then it should be commented out. Like here:
https://github.com/status-im/infra-role-nim-waku/blob/1e5367c86a906d9b08c0d492f5d70a85d790ae37/defaults/main.yml#L71-L74
The defaults file, aside from holding defaults, also plays the role of documentation, listing what can be configured.

Do you really want to have to read every single file in the role to find out what configuration options it supports?

**Comment by jakubgs** [comment]
Well whatever, I'll merge this and fix it myself, because I'm out till 11th and I won't be around to help you.

**Comment by jakubgs** [comment]
Amended: https://github.com/status-im/infra-role-nim-waku/commit/60ca2537

**Comment by jakubgs** [comment]
Also fixed the fleet repo commit: https://github.com/status-im/infra-nim-waku/commit/aa03f4bd

**Comment by jakubgs** [comment]
Deployed:
```
 > a all -a 'grep rln /docker/nim-waku-v2/docker-compose.yml'
node-01.do-ams3.wakuv2.test | CHANGED | rc=0 >>
      --rln-relay=true
      --rln-relay-content-topic='/toy-chat/2/luzhou/proto'
node-01.gc-us-central1-a.wakuv2.test | CHANGED | rc=0 >>
      --rln-relay=true
      --rln-relay-content-topic='/toy-chat/2/luzhou/proto'
node-01.ac-cn-hongkong-c.wakuv2.test | CHANGED | rc=0 >>
      --rln-relay=true
      --rln-relay-content-topic=/toy-chat/2/luzhou/proto
```

**Comment by staheri14** [comment]
Thanks @jakubgs for taking care of this PR. 

Despite all the recent updates, the rln-relay is not yet enabled on the waku v2 test fleets. 
Can it be due to this potential syntax error (there is an excess `-` after end if) https://github.com/status-im/infra-role-nim-waku/blob/1494bda34a38be9bfba83ec53e5e06ca98a4b7bb/templates/docker-compose.yml.j2#L55

**Comment by jakubgs** [comment]
>the rln-relay is not yet enabled on the waku v2 test fleets

It is deployed though:
```
 > a all -a 'grep rln /docker/nim-waku-v2/docker-compose.yml'
node-01.do-ams3.wakuv2.test | CHANGED | rc=0 >>
      --rln-relay=true
      --rln-relay-content-topic='/toy-chat/2/luzhou/proto'
node-01.gc-us-central1-a.wakuv2.test | CHANGED | rc=0 >>
      --rln-relay=true
      --rln-relay-content-topic='/toy-chat/2/luzhou/proto'
node-01.ac-cn-hongkong-c.wakuv2.test | CHANGED | rc=0 >>
      --rln-relay=true
      --rln-relay-content-topic=/toy-chat/2/luzhou/proto
```

**Pull request merged**